ViewGenerator/tools/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/webpack": "4.4.34",
+    "@types/webpack-manifest-plugin": "2.0.0"
+  },
+  "dependencies": {
```

Comment:
did you check the final webview* assembly, does it have only the necessary stuff?

Reply:
![Screenshot_4](https://user-images.githubusercontent.com/13751333/61808709-d9abf080-ae33-11e9-8e50-27ee9cf80e63.png)

Reply:
Seems to have only the necessary stuff

Reply:
And node_modules of Example projects seems only to contain the previous @types stuff from react and viewframe